packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/service.js

```diff
@@ -4,6 +4,33 @@ export default class OctaviaLoadBalancerListenersService {
     this.$http = $http;
   }
 
+  createL7Policy(
+    projectId,
+    region,
+    listenerId,
+    name,
+    position,
+    action,
+    redirectHttpCode,
+    redirectPoolId,
+    redirectPrefix,
+    redirectUrl,
+  ) {
```

Comment:
```suggestion
  createL7Policy(
    projectId,
    region,
    listenerId,
    {
      name,
      position,
      action,
      redirectHttpCode,
      redirectPoolId,
      redirectPrefix,
      redirectUrl,
    }
  ) {
```
if you use `model` as an argument in the controller